tokenizer.py

```diff
@@ -9,11 +9,17 @@
 import string
 
 def tokenize(imported_string):
-    # remove punctuation and empty strings
-    imported_string = imported_string.translate(str.maketrans('','',string.punctuation))
+    punctuation = '!"#$%&()*+,-./:;<=>?@[\\]^_`{|}~'  # same as string.punctuation, but allows for apostrophes
```

Comment:
What about non-english punctuation? Specifically thinking about things like `¿`, `« »`, and `„`?